unstructured/utils.py

```diff
@@ -354,6 +354,7 @@ def calculate_shared_ngram_percentage(
     n: int,
 ) -> tuple[float, set[tuple[str, ...]]]:
     """Calculate the percentage of common_ngrams between string_A and string_B
+
```

Comment:
Okay, blank line, good, but the first line needs to stand on its own, you can't just put the blank line in the middle of a sentence :)

Something like:
```python
"""Percentance of common-ngrams between string_A and string_B.

Common n-grams are computed with reference to ...
"""